setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='NotamMyWatch',
+    version='0.1.0',
```

Comment:
now that we have a version specification, we should have a team-wide discussion on:

1. When does this version get incremented?  Should ever PR be incrementing the version since it technically making a "new version" of the application with different features/functionality?
2. Should we start using Git tags at certain points to easily "pin" different points in time and versions of the application for easy identification?